README.md

```diff
@@ -457,6 +460,27 @@ Name        | Description                               | Risk Level | Token Req
 To see detailed information about each check, its scoring criteria, and
 remediation steps, check out the [checks documentation page](docs/checks.md).
 
+## Other Important Recommendations
+
+### Two-factor Authentication (2FA)
+
+The [2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/about-two-factor-authentication) is an extra layer of security when logging into websites or apps. With 2FA, you have to log in with your username and password and provide another form of authentication that only you know or have access to.
+
+It is a strong recommendation not only on Github but also in other website to always enable 2FA if avaiable. This way you will secure your account and, consequently, secure all the projects you contribute to.
```

Comment:
Suggested to replace the 469-471:

We strongly recommend that you enable 2FA on GitHub and any important account where it is available. 2FA is not a Scorecard check because GitHub does not make that data about user accounts public. Arguably, this data should always remain private, since accounts without 2FA are so vulnerable to attack. 

Though it is not an official check, we urge all project maintainers to enable 2FA to protect their projects from compromise.

Reply:
Done